testkit/src/main/java/org/apache/calcite/test/catalog/MockCatalogReaderSimple.java

```diff
@@ -261,6 +261,14 @@ protected MockCatalogReaderSimple(RelDataTypeFactory typeFactory,
     productsTable.addColumn("SUPPLIERID", fixture.intType);
     registerTable(productsTable);
 
+    // Register "EMPTY_PRODUCTS" table.
+    MockTable emptyProductsTable = MockTable.create(this, salesSchema, "EMPTY_PRODUCTS",
+        false, 2000D, 0.0);
```

Comment:
Is it intentional to have rows > masRows?

Reply:
It was intentional but fixed it anyway in the latest commit.